yarn-project/Earthfile

```diff
@@ -220,11 +221,16 @@ end-to-end:
     ENV ACVM_WORKING_DIRECTORY=/usr/src/acvm
     ENV ACVM_BINARY_PATH=/usr/src/noir/noir-repo/target/release/acvm
     ENV PROVER_AGENT_CONCURRENCY=8
-    RUN mkdir -p $BB_WORKING_DIRECTORY $ACVM_WORKING_DIRECTORY
+    RUN mkdir -p $BB_WORKING_DIRECTORY $
```

Comment:
What's `$` here?

Reply:
Looks like this got left off in a copy-paste.

Reply:
Thanks for pointing out.